### PR TITLE
BUG#1497765 - External SaaS endpoint integration is misleading -fixes

### PR DESCRIPTION
**Pull request by gaurav-nelson**
For https://bugzilla.redhat.com/show_bug.cgi?id=1497765
Updates docs to include provided info and fixed formatting issues

### CODE REVIEW

File: dev_guide/integrating_external_services.adoc [hunk 1]
```diff
@@ -346,17 +343,19 @@ use the service by setting environment variables in the appropriate containers:
...
 
 ----
-
 <1> Other fields on the `DeploymentConfig` are omitted.
 <2> `SAAS_API_KEY`: The API key to use with the service.
 <3> `SAAS_USERNAME`: The user name to use with the service.
 <4> `SAAS_PASSPHRASE`: The passphrase to use with the service.
 
++
+These variables gets added to the containers as environment variables. Using environment variable allows service-to-service communication and it may or may not require additional parameters such as API keys, user name and password authentication, or certificates.
```
Comment (gaurav-nelson) [inline dev_guide/integrating_external_services.adoc]: s/gets added/get added

**Comment by gaurav-nelson** [comment]
peer review please @openshift/team-documentation 

A small one :small_orange_diamond:

File: dev_guide/integrating_external_services.adoc (hunk 1 above)
Comment (ahardin-rh) [reply dev_guide/integrating_external_services.adoc]: s/variable/variables

**Comment by ahardin-rh** [comment]
@gaurav-nelson Just one extra comment from me ☪️

**Comment by gaurav-nelson** [comment]
Thanks @ahardin-rh :sunglasses:

**Comment by gaurav-nelson** [comment]
minor changes no revision history needed

**Pull request merged**